tests/conftest.py

```diff
@@ -60,6 +62,12 @@ def testing_data() -> Dataset:
     return data
 
 
+@pytest.fixture(scope="module")
+def dask_client() -> distributed.Client:
+    with Distributed(cluster_type="cpu") as dist:
```

Comment:
we can remove the cluster_type here if the the [gpu detection logic (currently relying on a numba import) is updated to something else](https://github.com/NVIDIA-Merlin/core/blob/8a23dc89b6b6b195670e1fea4162f506847c056b/merlin/core/utils.py#L33-L36)

Reply:
Whats the issue with the current GPU detection logic? Can we use the HAS_GPU flag from merlin.core.dispatch ?

https://github.com/NVIDIA-Merlin/core/blob/8a23dc89b6b6b195670e1fea4162f506847c056b/merlin/core/dispatch.py#L43-L45

Reply:
I've created a PR on core to try to make the automatic GPU detection logic inside this `merlin.core.utils.Distributeed` class more reliable here: https://github.com/NVIDIA-Merlin/core/pull/98